lib/govuk_schemas/random_example.rb

```diff
@@ -40,93 +40,70 @@ def self.for_schema(schema_key_value)
     # Example:
     #
     #      random = GovukSchemas::RandomExample.for_schema("detailed_guide", schema_type: "frontend")
```

Comment:
Should we make `for_schema` return the hash directly? We no longer need an object to talk to if it passes on the hash.

Reply:
I can do. I had left that method so that existing implementation using the easy solution would not need to be changed.

Reply:
Done